app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -104,7 +104,14 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="history">历史</string>
     <string name="sync">同步</string>
     <string name="search_or_custom">搜索 / 自定义</string>
-    <string name="save">保存</string>
+
```

Comment:
translations are managed in Crowdin <https://crowdin.com/project/markor/invite>. Please don't add translated strings in pull requests. Instead add the (chinese) translation then when the PR is merged ... at Crowdin.

--> remove the both changes to zh strings.xml

thanks

Reply:
ok